netbench/cdk/lambda/exportS3.js

```diff
@@ -1,19 +1,21 @@
 const AWS = require('aws-sdk')
 const cloudconfig = {
   apiVersion: '2014-03-28',
-  region: process.env.REGION, 
+  region: process.env.AWS_REGION, 
 }
 const cloudwatchlogs = new AWS.CloudWatchLogs(cloudconfig)
 exports.handler =  async (event, context) => {
    const params = {
     destination: process.env.BUCKET_NAME, 
-    from: new Date().getTime() - 600000, //TODO: will be replaced with the timestamp of the start of statemachine execution
+    from: new Date().getTime() - 6000000, //TODO: will be replaced with the timestamp of the start of statemachine execution
```

Comment:
why is this changing?

Reply:
The value there is still a placeholder, I haven't gotten around to adjust that yet. Essentially, I was running tests for longer than 10 minutes(600000 ms) so I made it larger.